src/napari_vedo_bridge/_cutter_widget.py

```diff
@@ -35,32 +33,39 @@ def __init__(self, napari_viewer=None):
         self.box_cutter_widget = None
         self.sphere_cutter_widget = None
 
+        self.vedo_message = Text2D(font='Calco', c='white')
+
     def get_from_napari(self):
         """
         Get the currently selected layer from napari and display it in vedo
         """
-
         self.currently_selected_layer = self.napari_viewer.layers.selection.active
+        
         self.mesh = Mesh(self.currently_selected_layer.data[:2])  # only vertices and faces
+        
+        # self.mesh = Mesh("997.ply")
+        # self.mesh = Mesh("beethoven.ply")
+
+        self.mesh.triangulate()
+        self.mesh.c("yellow5").backcolor("purple6").lighting("glossy")
+
         #mesh.pointdata = self.currently_selected_layer.features  # scalars
         # Create renderer and add the vedo objects and callbacks
 
         self.pushButton_box_cutter.setChecked(False)
         self.pushButton_sphere_cutter.setChecked(False)
         self.pushButton_plane_cutter.setChecked(False)
 
-        if len(self.plt.actors) > 0:
```

Comment:
I think this we should keep. If I try it now and cut the same mesh multiple times, I keep aggregating actors in the cutting widget.